src/search/engines/bing/bing.go

```diff
@@ -70,28 +60,39 @@ func Search(ctx context.Context, query string, relay *bucket.Relay, options engi
 		}
 	})
 
-	localeParam := getLocale(&options)
+	localeParam := getLocale(options)
 
+	// create a buffered channel to hold errors, size of channel is the number of pages = requests being made
+	errChannel := make(chan error, options.MaxPages)
```

Comment:
This should either also catch the errors from the colly callbacks, or shouldn't be a channel and should just be a slice. There is no reason for it to be a channel as opposed to a slice if its used in a sync context (which it currently is).
See this for modyfing slice in `Do(Get/Post)Request`: https://stackoverflow.com/questions/49428716/pass-slice-as-function-argument-and-modify-the-original-slice